src/testHelpers/mockComponents.jsx

```diff
@@ -6,7 +6,11 @@ export const ComponentUsingContext = ({ context }) => {
   const value = useContext(context);
   return (
     <>
-      <span>{JSON.stringify(value)}</span>
+      <span>
+        {JSON.stringify(value, (key, val) =>
+          typeof val === 'function' ? `*function - ${val.name}*` : val,
```

Comment:
struggling to understand what's happening here

Reply:
I was going to leave a comment in the code about this but ended up forgetting, my bad.
In the `withContexts.test.jsx`, I had some strange snapshot errors as seen below:
![image](https://user-images.githubusercontent.com/53564281/66197582-94f1d080-e692-11e9-80f8-1baa3c17a069.png)
This is because `JSON.stringify` returns functions as `null`. The replacer `(key, val)` allows `JSON.stringify` to replace `null` with an identify-able string using the function name